src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue9796.cs

```diff
@@ -0,0 +1,31 @@
+﻿#if ANDROID
```

Comment:
Remove this so that the test runs on all three platforms

Reply:
condition removed